Solutions/GitHub/Hunting Queries/Unusual Number of Repository Clones.yaml

```diff
@@ -0,0 +1,24 @@
+id: ccef3c74-4b4f-445b-8109-06d38687e4a4
+name: GitHub Repo Clone - Time Series Anomly
+description: |
+  'Attacker can exfiltrate data from you GitHub repository after gaining access to it by performing clone action. This hunting queries allows you to track the clones activities for each of your repositories. The visualization allow you to quickly identify anomalies/excessive clone, to further investigate repo access & permissions'
+requiredDataConnectors: []
```

Comment:
Please add the connectorid and datatype in this format
requiredDataConnectors: -
connectorId: connector id
dataTypes: - name of the parser which we use in the query

Thanks,
Aditi Maheshwari

Reply:
Added